config/backend-config/namespace_config.go

```diff
@@ -154,7 +154,9 @@ func (nc *namespaceConfig) getFromAPI(ctx context.Context, _ string) (ConfigT, e
 		}
 		sourcesJSON.Sources = append(sourcesJSON.Sources, nc.Sources...)
 	}
-
+	sourcesJSON.ConnectionFlags.URL = nc.cpRouterURL
+	// always set connection flags to true for hosted and multi-tenant warehouse service
+	sourcesJSON.ConnectionFlags.Services = map[string]bool{"warehouse": true}
```

Comment:
I don't see a test in `namespace_config_test.go` about this change. Can you please add it?

Reply:
Added the test